### DISCUSSION TITLE
select map layers to display

### DISCUSSION THREAD

**Discussion by frits44** [organicmaps/organicmaps#744] [Ideas]
Hello,
I really like Organic Maps, but there is one thing that irritates me and sometimes makes it hard to find an address:
The layer with shops and other commercial "stuff". Would it be possible to make certain layers selectable? In a similar other app all layers are selectable. All layers may be overkill, but I like to have my map clean.

**Comment by biodranik** [comment]
Hi, it sounds like a good idea. What are your usual use-cases for it? Why exactly do you need a clean map?

**Comment by frits44** [comment]
Thank you for the quick reply. 
I (we) visit people and often have to really search. I am not interested in commercial elements, especially because they are often irrelevant and/or outdated. House-numbers are very important for us. The possibility to remove POI's (icons and lables) would be fine.
